internal/plugins/envtest/v1/plugin.go

```diff
@@ -29,7 +28,7 @@ const pluginName = "envtest" + plugins.DefaultNameQualifier
 
 var (
 	pluginVersion            = plugin.Version{Number: 1}
-	supportedProjectVersions = []config.Version{cfgv2.Version, cfgv3.Version}
+	supportedProjectVersions = []config.Version{cfgv3.Version}
```

Comment:
Asked this elsewhere, but does this plugin need to remain inf we're deprecating all internal uses for it?

Reply:
probably not. makes sense to remove.